config_test.go

```diff
@@ -25,3 +29,23 @@ func TestLoadBadConfig(t *testing.T) {
 		t.Errorf("\nexpected:\n%v\ngot:\n%v", expected, err.Error())
 	}
 }
+
+func TestPrometheusTimeout(t *testing.T) {
+	initConfig()
+
+	req, err := http.NewRequest("GET", "/probe?module=http_2xx&target=http%3A%2F%2Fprometheus.io", nil)
```

Comment:
We shouldn't presume internet access, see the http tests for how to setup a test server